data/models/change.py

```diff
@@ -1,202 +1,132 @@
-    """🔀 Pour pointer un acteur vers un parent,
-    on MAJ sa data, donc on part du modèle d'update
-    et on ajoute juste une validtion sur data"""
-
-    def data_validate(self):
-        # Si on doit rediriger un acteur vers un parent
-        # alors on s'attend à ce que le parent existe
-        parent = RevisionActeur.objects.filter(
-            identifiant_unique=self.data["parent_id"]
-        )
-        if not parent.exists():
-            raise ValueError(f"Parent {self.data['parent_id']} non trouvé")
-
-
-class ChangeActeurCreateAsParent(ChangeActeurCreateRevision):
-    """➕ Pour la création d'un parent, on créer une révision,
-    donc on part du modèle de révision et on ajoute juste
-    une validation data"""
-
-    def data_validate(self):
-        # Si le parent est a créer, alors on s'attend à ce qu'il n'existe pas
-        rev = RevisionActeur.objects.filter(identifiant_unique=self.identifiant_unique)
-        if rev.exists():
-            raise ValueError(f"Parent à créer {self.identifiant_unique} déjà existant")
-
-
-class ChangeActeurParentKeep(ChangeActeurUpdateData):
-    """🟢 Pour la MAJ d'un parent existant, on le met à jour
-    comme on MAJ un acteur mais avec la data qui nous intéresse"""
-
-    def data_validate(self):
-        # On fait confiance à la donnée qu'on reçoit,
-        # on laisse les modèles Django se plaindre si pas content
-        # MAIS on vérifie juste qu'on ne vient pas donner de source
-        # à un parent
-        if "source" in self.data or "source_id" in self.data:
-            raise ValueError("Pas de source pour un parent")
-
-
-class ChangeActeurParentDelete(ChangeActeurDeleteRevision):
-    """🔴 Pour la suppression d'un parent, on supprime sa révision,
-    donc on part du modèle de revision et on ajoute juste
-    une validation data"""
-
-    def data_validate(self):
-        # Si notre logique de suppression automatique
-        # des parents sans enfants fonctionne, alors
-        # quand on arrive ici, le parent doit avoir été supprimé
-        # par les observers
-        rev = RevisionActeur.objects.filter(identifiant_unique=self.identifiant_unique)
-        if rev.exists():
-            raise ValueError(
-                f"Parent {self.identifiant_unique} devrait déjà être supprimé"
-            )
-
-
-# -----------------------------
-# 3) Mapping change_type ➡️ modèle
-# -----------------------------
-MAPPING_TYPES_TO_MODELS = {
-    # -----------------------------
-    # Changements de type clustering/déduplication
-    # -----------------------------
-    CHANGE_ACTEUR_POINT_TO_PARENT: ChangeActeurPointToParent,
-    CHANGE_ACTEUR_CREATE_AS_PARENT: ChangeActeurCreateAsParent,
-    CHANGE_ACTEUR_PARENT_KEEP: ChangeActeurParentKeep,
-    CHANGE_ACTEUR_PARENT_DELETE: ChangeActeurParentDelete,
-}
+        order = self.order
+        reason = self.reason
+        model_name = self.model_name
+        model_params = self.model_params
+        Model = CHANGE_MODELS[model_name]
```

Comment:
Est-ce qu'on peut spécifier le type de `model` utilisé ? Par exemple 
```suggestion
        ChangeModel = CHANGE_MODELS[model_name]
```

Ou 

```suggestion
        SuggestionChangeModel = CHANGE_MODELS[model_name]
```